website/docs/cli/commands/graph.mdx

```diff
@@ -37,8 +37,7 @@ Options:
 * `-draw-cycles`    - Highlight any cycles in the graph with colored edges.
   This helps when diagnosing cycle errors.
 
-* `-type=plan`      - Type of graph to output. Can be: `plan`, `plan-destroy`, `apply`,
-  `validate`, `input`, `refresh`.
+* `-type=plan`      - Type of graph to output. Can be: `plan`, `plan-refresh-only`, `plan-destroy`, or `apply`.
```

Comment:
Hello, @cmaritan! 👋 I apologize for the late review here, but thank you for catching this! In the interest of getting this in quickly, I made a small suggestion for style above and am going to commit it, so I can review the deploy preview and get this merged asap!